gradle/libs.versions.toml

```diff
@@ -51,6 +51,8 @@ guava = "33.4.6-jre"
 hadoop3 = "3.4.1"
 httpcomponents-httpclient5 = "5.4.3"
 hive2 = { strictly = "2.3.10"} # see rich version usage explanation above
+hive3 = { strictly = "3.1.3"} # see rich version usage explanation above
```

Comment:
We need LICESE and NOTICE checks before committing this since there could be updates that we need to include.

Reply:
I'm not familiar with this process. Is there something I need to run? Or is this a manual check of the LICENSE and NOTICE files?

Reply:
I checked the LICENSE and NOTICE files. I updated the LICENSE file. The NOTICE file does not need any update.